packages/l/libusb/xmake.lua

```diff
@@ -5,18 +5,41 @@ package("libusb")
 
     set_urls("https://github.com/libusb/libusb/archive/$(version).tar.gz",
              "https://github.com/libusb/libusb.git")
+    add_versions("v1.0.24", "b7724c272dfc5713dce88ff717efd60f021ca5b7c8e30f08ebb2c42d2eea08ae")
     add_versions("v1.0.23", "02620708c4eea7e736240a623b0b156650c39bfa93a14bcfa5f3e05270313eba")
 
     if not is_host("windows") then
         add_deps("autoconf", "automake", "libtool", "pkg-config")
     end
 
-    add_includedirs("include/libusb-1.0")
```

Comment:
> What was the purpose of the add_includedirs directive? From what I saw in debian packages you should include libusb-1.0/libusb.h to use it.

export includedirs to user projects, user may will use `#include <libusb.h>`

we can use this:

```lua
    add_includedirs("include", "include/libusb-1.0")
```

Reply:
Why do we do it for this library and not other?
From what I see in the debian package, it's installed in a libusb-1.0 folder, which means #include <libusb.h> isn't valid.
```
:~$ dpkg -L libusb-1.0-0-dev
/.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/libusb-1.0-0-dev
/usr/share/doc/libusb-1.0-0-dev/copyright
/usr/lib
/usr/lib/x86_64-linux-gnu
/usr/lib/x86_64-linux-gnu/libusb-1.0.a
/usr/lib/x86_64-linux-gnu/pkgconfig
/usr/lib/x86_64-linux-gnu/pkgconfig/libusb-1.0.pc
/usr/include
/usr/include/libusb-1.0
/usr/include/libusb-1.0/libusb.h
/usr/share/doc/libusb-1.0-0-dev/README
/usr/share/doc/libusb-1.0-0-dev/changelog.Debian.gz
```

Reply:
It's official examples: https://github.com/libusb/libusb/blob/a108bb8e185cc5c41b296d35dfa83e5e5b13b2c0/examples/listdevs.c#L22

and homebrew 

https://github.com/Homebrew/linuxbrew-core/blob/27ce9737e78d9144910f14d6b15d2ff4055915a4/Formula/libusb.rb#L43

Many users may use `<libusb.h>`

`add_includedirs("include", "include/libusb-1.0")` can support `<libusb.h> and <libusb-1.0/libusb.h>`

Reply:
Alrighty then